model/qti/ImportService.php

```diff
@@ -94,6 +94,8 @@ class ImportService extends ConfigurableService
 
     public const PROPERTY_QTI_ITEM_IDENTIFIER = 'http://www.tao.lu/Ontologies/TAOItem.rdf#QtiItemIdentifier';
 
+    private const MEDIA_ROOT_CLASS = 'http://www.tao.lu/Ontologies/TAOMedia.rdf#Media';
```

Comment:
This const live here: `tao/taoMediaManager/model/TaoMediaOntology.php:27`
But it is in the MediaManager namespace. This raises a concern that it is not a proper place for this code. Media are not a part of itemQti extension and it should not be handle here

Reply:
This constant was used in the process anyway I just moved it in another place. The issue of coupling media manager is bigger than just this const, to decouple it correctly we need a huge refactoring. Other option is to give up and consider it as a dependency. To be precise this PR is more a fix than a feature.